src/main.go

```diff
@@ -34,19 +48,41 @@ type QueryParameter struct {
 var katsuQueryConfigCache = make(map[string]interface{})
 
 func main() {
+	// Initialize configuration from environment variables
 	var cfg BentoConfig
 	err := envconfig.Process("", &cfg)
 	if err != nil {
 		fmt.Println(err)
 		os.Exit(2)
 	}
 
-	fmt.Println(fmt.Sprintf(`Config --
-		Client Name: %s
-		Katsu URL: %v
-		Maximum no. Query Parameters: %d
-		Bento Portal Url: %s
-	`, cfg.ClientName, cfg.KatsuUrl, cfg.MaxQueryParameters, cfg.BentoPortalUrl))
+	// Load JS package.json to extract version number
+	packageJson, err := ioutil.ReadFile(cfg.PackageJsonPath)
+	if err != nil {
+		fmt.Println("Error reading package.json")
+		os.Exit(1)
+	}
+
+	var packageJsonContents map[string]interface{}
+	err = json.Unmarshal(packageJson, &packageJsonContents)
+	if err != nil {
+		fmt.Println("Error parsing package.json")
+		os.Exit(1)
+	}
+
+	var version = fmt.Sprintf("%s", packageJsonContents["version"])
```

Comment:
Disclaimer: I'm a golang noob. Why use fmt.Sprintf here if the returned string is equal to `packageJsonContents["version"]` ?

Reply:
I thought I had a reason at some point, but I think it must have been an issue elsewhere triggering an IDE warning - anyway I'll remove it since you're right, it's redundant